changelog/v1.0.0-beta14/access-log-filterstate.yaml

```
@@ -0,0 +1,5 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo-mesh-enterprise/issues/165
+    resolvesIssue: false
+    description: Allow access logs to capture additional data set by envoy filters, using filterStateObjects.
```

```suggestion
    description: Allow access logs to capture additional data set by envoy filters, filtering by state.
```
^ Maybe clarify what 'state' means, exactly here. E.g. filtering by the returned status code of the request?

---

Good point, I can clarify. Filter State Objects are an Envoy construct that describe arbitrary state from the envoy filters. I'll update the description.